Dockerfile

```diff
@@ -0,0 +1,52 @@
+# python:3-alpine contains node 18 so has to go first
+# in order to get overwritten
+FROM python:3-alpine
+FROM node:14-alpine
```

Comment:
And from a similar PR on site-community: the feedback there, which I shared as well, is that this would probably be better off as a Ubuntu image, as the team has more experience with Ubuntu and it's our standard elsewhere.

The double `FROM` is also a point that came up in the other pull request. I haven't seen this either. From the docs, it's not really clear if this creates a single image or multiple images:

> FROM can appear multiple times within a single Dockerfile to create multiple images or use one build stage as a dependency for another. Simply make a note of the last image ID output by the commit before each new FROM instruction. Each FROM instruction clears any state created by previous instructions.

Reply:
I agree, the multiple `FROM` is not desirable.

I'll try something based on a minimal Ubuntu...

Reply:
If the double FROM works, that's okay, I think we're just not familiar enough to :+1: it

Reply:
Hm, the Ubuntu image will require either running `su -c 'curl -sL https://deb.nodesource.com/setup_14.x | bash -'` or otherwise some NVM stuff. I think it's better to use the offical Node images and put Python 3 in them. They also have Debian images.

Reply:
Seems like a good place to use `asdf` actually.

Reply:
Understanding now how multiple `FROM` commands work! The former pattern was wrong, so need to be more explicit. Have included a comment for now that 3.10 is implied by `node:14-alpine`. Will figure out something that's easier to maintain.